app/assets/stylesheets/lead_providers/statements.scss

```diff
@@ -54,6 +54,10 @@
   p {
      display: flex;
      justify-content: space-between;
+
+     span {
+       margin-left: 15px;
```

Comment:
the space between `Total net VAT` and the value might need a bit of spacing on normal views (tested locally)